src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -213,8 +213,8 @@ function SplitExplicitAuxiliaryFields(grid::AbstractGrid)
     fill_halo_regions!((Hᶠᶜ, Hᶜᶠ))
 
     kernel_parameters = :xy
-    
-    return SplitExplicitAuxiliaryFields(Gᵁ, Gⱽ, Hᶠᶜ, Hᶜᶠ, kernel_parameters)
+
+  return SplitExplicitAuxiliaryFields(Gᵁ, Gⱽ, Hᶠᶜ, Hᶜᶠ, kernel_parameters)
```

Comment:
```suggestion
    return SplitExplicitAuxiliaryFields(Gᵁ, Gⱽ, Hᶠᶜ, Hᶜᶠ, kernel_parameters)
```

The indent was right. It's the rest of the function that's wrong.